src/App.tsx

```diff
@@ -38,6 +38,21 @@ export const App = () => (
             <div className="ContentWrapper">
               <Header />
 
+              <div className="Announcement">
+                <p>
+                  Welcome to the new and improved Stellar demo wallet! Please
+                  log bugs and feature requests at: &nbsp;
+                  <TextLink
+                    variant={TextLinkVariant.primary}
+                    href="https://github.com/stellar/stellar-demo-wallet/issues"
+                    rel="noreferrer"
+                    target="_blank"
+                  >
+                    https://github.com/stellar/stellar-demo-wallet/issues
+                  </TextLink>
+                </p>
```

Comment:
To have consistent width and horizontal padding, we need to wrap `<p>` in `<div className="Inset">`